cdap-ranger/cdap-ranger-common/src/main/java/co/cask/cdap/security/authorization/ranger/commons/RangerCommon.java

```diff
@@ -35,6 +35,6 @@
   public static final String KEY_PRINCIPAL = "principal";
 
   // using # as we don't allow it in entity names
-  public static final String RESOURCE_SEPARATOR = "#";
+  public static final String RESOURCE_SEPARATOR = ".";
```

Comment:
Ya it just for better consistency since ranger internally saves the parts as Map so having a . in value will not affect comparison and its more consistent with entity representation.